pool/src/pool/pool.rs

```diff
@@ -47,6 +49,19 @@ impl Pool {
         } else {
             Reserve::load(e, &self.config, asset)
         }
+    });
+
+    pub(crate) fn load_reserve_spec(&mut self, _e: &Env, asset: &Address, store: bool) -> Reserve {
```

Comment:
should this be in the certora crate?

Reply:
also i think a comment might be nice about `apply_summary!`?

Reply:
I have this one here because it's a method on `Pool`, so I can't define it outside of the `impl` block here (this is in general a problem if you want to use private fields in the summary)

Reply:
oh oh sorry, you are right.